scenarios/17_Sneaking_out_of_Raedwood.cfg

```diff
@@ -571,6 +582,17 @@
         [/objectives]
     [/event]
 
+    [event]
+        name=recall
+        [filter]
+            # wmllint: recognize Karl Regven
+            id=Karl Regven
+        [/filter]
+        {MSGm_Gawen _"Wait... Karl... why aren't you guarding Karen?"}
```

Comment:
Ellipses can be commas instead.

Reply:
changed in [efba779](https://github.com/nemaara/A_New_Order/pull/43/commits/efba77972af8c9bcc1fc309bcc70c2a6748d2016)